src/Microsoft.AspNet.Mvc.Abstractions/ModelBinding/ModelBindingResult.cs

```diff
@@ -9,6 +9,16 @@ namespace Microsoft.AspNet.Mvc.ModelBinding
     public class ModelBindingResult
     {
         /// <summary>
+        /// Creates a new <see cref="ModelBindingResult"/> indicating a fatal error.
+        /// </summary>
+        /// <param name="key">The key using which was used to attempt binding the model.</param>
+        public ModelBindingResult(string key)
```

Comment:
In future PRs remove all `public` constructors and use explicit `static` factory methods instead.